documentation/modules/ROOT/pages/index.adoc

```diff
@@ -81,24 +75,19 @@ xref:python-sdk-guide:ROOT:index.adoc[Python SDK]
 [.icon-cli]
 --
 [.title]
-JS SDK Aspect Model Loader
+JavaScript SDK
 
 [.text]
-The JS SDK Aspect Model Loader offers functionality which helps software developers to work with Aspect Models 
-in their Typescript applications.
+Contains several parts – all aimed at creating a graphical user interface (GUI) for your solution, based on Aspect Models.
```

Comment:
Create UIs better and faster. This SDK loads Aspect Models and creates UI code e.g. tables that unite Aspect data & model.

Reply:
Done with https://github.com/bci-oss/eclipse-esmf.github.io/pull/4/commits/dfce54eb88112f18454462deae08fc5fff053e18 ✔